fusor-ember-cli/app/components/base-f.js

```diff
@@ -16,11 +16,11 @@ export default Ember.Component.extend({
   },
 
   labelClassSize: Ember.computed(function () {
-    return this.getWithDefault('labelSize', 'col-lg-2 col-md-3 col-sm-5');
+    return this.getWithDefault('labelSize', 'col-lg-3 col-md-3 col-sm-4 col-xs-4 col-xs-offset-1');
```

Comment:
Changing back sm to 5/7 would probably be a good idea just to make sure we don't accidentally modify other forms if we know they're working well and often on small screens we want to use all available real estate.

I think sensible defaults for base-f is good idea as long as they make sense system-wide.  I thought this looked like it was specific to that modal so it made more sense to me to be on that modal, but if these values make sense on any field as the default, then it's fine.  Up to you which you think is best.